Susi/Controllers/ChatViewController/ChatVCMethods.swift

```diff
@@ -340,7 +340,6 @@ extension ChatViewController {
             let lastItem = messages.count - 1
             let indexPath = IndexPath(item: lastItem, section: 0)
             collectionView?.scrollToItem(at: indexPath, at: .top, animated: true)
-            scrollButton.isHidden = true
```

Comment:
When we scroll the chat directly instead of pressing scroll button to scroll, the scroll button is still there.

Reply:
@jogendra I just rechecked, this will happen even when`scrollButton.isHidden = true` command stays. I am planning to work on that issue separately. 😅